### PR TITLE
show pending mortgages

### PR DESCRIPTION
**Pull request by nachomazzara**
Remove the constraint to not show pending mortgages if the publication is closed to allow accounts to refund the deposit made when requested a mortgage

### CODE REVIEW

File: src/Translation/locales/en.json [hunk 1]
```diff
@@ -280,17 +280,21 @@
       "deposit_gt_balance": "Required deposit is greater than your balance",
       "duration_gte_payable":
         "Duration should be greater or equal than payable at days",
+      "insufficient_funds": "You don't have anought MANA. Required: {amount}",
```
Comment (cazala) [inline src/Translation/locales/en.json]: **e**nough

File: src/Translation/locales/es.json [hunk 2]
```diff
@@ -280,6 +280,7 @@
       "deposit_gt_balance": "El depósito requerido es mayor que su saldo",
       "duration_gte_payable":
         "La duración debe ser mayor o igual que la paga en días",
+      "insufficient_funds": "No tienes nada MANA. ",
```
Comment (cazala) [inline src/Translation/locales/es.json]: No tienes suficiente MANA

File: src/Translation/locales/fr.json [hunk 3]
```diff
@@ -286,6 +286,7 @@
       "deposit_gt_balance": "Le dépôt requis est supérieur à votre solde",
       "duration_gte_payable":
         "La durée devrait être supérieure ou égale à payable à des jours",
+      "insufficient_funds": "Vous n'avez pas nought2. ",
```
Comment (cazala) [inline src/Translation/locales/fr.json]: I dunno french but I don't think "nought2" is a word :P

File: src/Translation/locales/en.json (hunk 1 above)
Comment (nachomazzara) [reply src/Translation/locales/en.json]: uhhhhhhhhhh nice catch

**Pull request merged**